demo/00Index

```diff
@@ -1,2 +1,3 @@
 example_1		Basic package usage
 example_2       Using data.frame as input data
+example_3       LDA example
```

Comment:
Perhaps `pkg_overview`, `df_example`, `perform_lda`?

Reply:
No a bad idea.  Maube `rvw_overview`, `rvw_df_examples`, `rvw_lda_example` ?   `demo()` lists all, even though it is chunked it is nice if they 'look' similar.  My $0.02.

Reply:
yes, sounds really reasonable